
(25, 'Inappropriate ioctl for device')
[nltk_data] Downloading package punkt to
[nltk_data]     /cluster/home/julievt/nltk_data...
[nltk_data]   Package punkt is already up-to-date!
Traceback (most recent call last):
  File "/cluster/home/julievt/master-project/./scripts/train/tre/tlink/main.py", line 72, in <module>
    ner = TRExtract(
          ^^^^^^^^^^
  File "/cluster/home/julievt/master-project/src/textmining/tre/setup.py", line 87, in __init__
    test_dataset_tre = test_manager.get(task)
                       ^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
